specification/apicenter/DataApi/models/api.tsp

```diff
@@ -0,0 +1,120 @@
+namespace ApiCenter.DataApi.Models;
+
+@doc("API contract")
+model Api {
+  @doc("The resource name.")
+  @maxLength(50)
+  name: string;
+
+  @doc("The name of the API.")
+  @maxLength(50)
+  title: string;
+
+  @doc("Short descrption of the API.")
+  @maxLength(200)
+  summary?: string;
+
+  @doc("The description of the API")
+  @maxLength(1000)
+  description?: string;
+
+  @doc("Kind of API. For example, REST or GraphQL.")
+  kind: string;
+
+  @doc("Current lifecycle stage of the API.")
+  lifecycleStage?: ApiLifecycleStage;
+
+  @doc("Terms of service for the API.")
+  termsOfService?: ApiTermsOfService;
+
+  @doc("The license information for the API.")
+  license?: ApiLicenseModel;
+
+  @doc("External documentation")
+  externalDocumentation?: Array<ApiExternalDocumentation>;
```

Comment:
Use `ApiExternalDocumentation[]` syntax instead. Same for all arrays, or is this a custom type you defined I haven't gotten to yet?

Reply:
I see it's the built-in type. We recommend the `T[]` for readability.

Reply:
Fixed in latest revision.